packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -554,11 +556,12 @@ class IOSSimulator extends Device {
   String get sdkNameAndVersion => category;
 
   @override
-  DeviceLogReader get logReader {
-    if (_logReader == null)
-      _logReader = new _IOSSimulatorLogReader(this);
+  DeviceLogReader get logReader => logReaderForApp(null);
```

Comment:
Passing null materially affects the behavior of the log reader in that it causes the log reader to show logs from all apps.